user-registration-app/src/app/views/user-registration-view/user-registration-view.component.ts

```diff
@@ -71,130 +71,121 @@ export class UserRegistrationViewComponent implements OnInit {
 
   ngOnInit() {
     const ecObservable = this.httpService.getExtraCreditClasses()
-      .subscribe(classes => {
+      .subscribe((classes) => {
         this.classes = classes;
         ecObservable.unsubscribe();
       });
     const regObservable = this.httpService.getUserRegistrations()
-      .subscribe(registrations => {
-        this.registrations = registrations;
-        regObservable.unsubscribe();
-      },         ({ error }) => {
-        if (error.status === 404) {
-          this.alertsService.info('You have not registered for a hackathon yet. We could not find any data for those queries');
+      .subscribe((registrations: RegistrationApiResponse[]) => {
+        if (registrations) {
+          this.activeRegistration = registrations.filter(registration => registration.hackathon.active)[0];
         }
+        regObservable.unsubscribe();
       });
+      // The error for registrations gets handled in user-profile-view
```

Comment:
Apparently the user-profile-view component also gets loaded and also tries to get registrations. So if they both fail, two alerts pop up